pmm-app/src/pmm-dbaas/components/XtraDB/AddXtraDBModal/AddXtraDBModal.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FC, useState } from 'react';
+import { Field, Form } from 'react-final-form';
+import { HorizontalGroup } from '@grafana/ui';
+import { LoaderButton, TextInputField, validators } from '@percona/platform-core';
+import { Modal } from 'shared/components/Elements/Modal/Modal';
```

Comment:
You will be surprised, but I just discovered that the Modal component from `@grafana/ui` is no different from our own :) Could you please double check, so that I'm not mistaken. If that is true, we probably don't need our own - it's an unnecessary wrapper on that one.

Reply:
Our own at least adds one more feature that the one from Grafana doesn't have, which is close on escape. It was one of the requirements requested by you in [PMM-6568](https://jira.percona.com/browse/PMM-6568).